.java-version

```diff
@@ -0,0 +1 @@
+22
```

Comment:
My `jenv` doesn't have `22` as a version; it only has `22.0` and `22.0.2`. Maybe make this `22.0`?

Reply:
I have both in my `jenv`. Anyway, changed it to `22.0`.